.github/workflows/ci.yaml

```diff
@@ -6,34 +6,74 @@ on:
   push:
 
 jobs:
-  build-and-test:
+  cmake:
     strategy:
       matrix:
-        ROS_DISTRO: [humble, rolling]
+        ros_distro: [humble, rolling]
+        preset: [debug, release, asan, codecov]
     runs-on: ubuntu-22.04
-    container: ghcr.io/picknikrobotics/rsl:upstream-${{ matrix.ROS_DISTRO }}
+    container: ghcr.io/picknikrobotics/rsl:upstream-${{ matrix.ros_distro }}
     env:
       CCACHE_DIR: ${{ github.workspace }}/.ccache
     steps:
       - name: Check out the repo
         uses: actions/checkout@v3
       - name: Cache ccache
-        uses: pat-s/always-upload-cache@v3.0.1
+        uses: pat-s/always-upload-cache@v3
```

Comment:
Does this mean we always get the latest release in the 3.x series?

Reply:
Yes, I didn't see any reason to specify the patch version here.